### PR TITLE
RIA-3882 Admin officer journey chooses fee update not required after additional fee required

### PR DESCRIPTION
**Pull request by tarun-palisetty**
### JIRA link (if applicable) ###
[RIA-3882](https://tools.hmcts.net/jira/browse/RIA-3882)

### Change description ###
Admin officer journey chooses fee update not required after additional fee required

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by divisathyan — CHANGES_REQUESTED** [review]
Please look at the comments

File: src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ManageFeeUpdateMidEventTest.java [hunk 1]
```diff
@@ -95,6 +95,44 @@ void handling_should_error_if_fee_update_reason_is_not_present() {
             .hasMessage("New fee amount is not present");
     }
 
+    @Test
+    void handle_should_error_if_refund_approved_selected_after_additional_fee_requested() {
```
[the remark this answers is not in the document]
Comment (tarun-palisetty) [reply src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ManageFeeUpdateMidEventTest.java:99]: it is as per AC

[the remark this answers is not in the document]
Comment (divisathyan) [reply src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ManageFeeUpdateMidEventTest.java:99]: AC was modified. I have modified the test case

**Review by mkayad — APPROVED** [review]
LGTM

**Review by Srihari-Nakkala — DISMISSED** [review]
LGTM

**Review by jkumar-moj — APPROVED** [review]

**Review by divisathyan — APPROVED** [review]

**Pull request merged**